lib/puppet/functions/eyaml_lookup_key.rb

```diff
@@ -0,0 +1,73 @@
+# @since 4.9.0
```

Comment:
This is not true if this ends up going into Puppet 5.0.0. Need to wait for decision.